docs/node-mixin/dashboards/use.libsonnet

```diff
@@ -201,8 +201,8 @@ local g = import 'grafana-builder/grafana.libsonnet';
           { yaxes: g.yaxes('percentunit') },
         )
         .addPanel(
-          g.panel('Memory Saturation (pages swapped per second)') +
-          g.queryPanel('instance:node_memory_swap_io_pages:rate1m{%(nodeExporterSelector)s, instance="$instance"}' % $._config, 'Swap IO') +
+          g.panel('Memory Saturation') +
+          g.queryPanel('instance:node_vmstat_pgmajfault:rate1m{%(nodeExporterSelector)s, instance="$instance"}' % $._config, 'Major page faults') +
```

Comment:
Here we have the hint in the legend, that's probably fine. On the other hand, we had a hint in the panel title before. So perhaps add it as suggested for the other panel above.

Reply:
Added here too.